siliconcompiler/tools/openroad/scripts/sc_floorplan.tcl

```diff
@@ -314,16 +324,9 @@ if {[dict exists $sc_cfg constraint component]} {
     set x_loc [expr round($x_loc / $x_grid) * $x_grid]
     set y_loc [expr round($y_loc / $y_grid) * $y_grid]
 
-    set place_args []
-    if { $cell != "" } {
-      lappend place_args "-cell" $cell
-    }
-
-    place_cell -inst_name $name \
-      -origin "$x_loc $y_loc" \
-      -orient [$transform_final getOrient] \
-      -status FIRM \
-      {*}$place_args
+    $inst setOrient [$transform_final getOrient]
```

Comment:
So before we placed a cell but now we place an instance of said cell?
So cells don't have to be instanciated first?

Reply:
the place_cell code doesn't work right, so I'm juts bypassing it.